nette/notejam/app/presenters/SecuredBasePresenter.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Presenters;
+
+
+class SecuredBasePresenter extends BasePresenter
```

Comment:
Oh, thank you! It seems I hacked the application too fast, missed some things. Silly me.